lib/lib.go

```diff
@@ -743,3 +744,20 @@ func (inst *Instance) RemoteClient() remote.Client {
 func (inst *Instance) Teardown() {
 	inst.teardown()
 }
+
+// checkRPCError validates RPC errors and in case of EOF returns a
+// more user friendly message
+func checkRPCError(err error) error {
+	if strings.Contains(err.Error(), "EOF") {
+		msg := `Qri wasn't able to execute your command. 
+It is possible you have a Qri node running or the Desktop app is open.
+Try closing them and running the command again.
+Check our issue tracker for RPC issues & feature requests:
+  https://github.com/qri-io/qri/issues?q=is:issue+label:RPC
+
+Error:
+%s`
```

Comment:
Language of this looks great. 

I just kinda hacked together this convention in the SQL pr, does it work for you? More info we could include?

Reply:
Yeah I saw that back in the PR and thought it was a bit ellaborate but very much user friendly. 
Just a couple bits on the language itself:
Qri actually does execute the command but couldn't parse the response (and we don't know if it was successful or not)! maybe should change to this.
Other than that, I'd just like more "specific" error handling and not just a blanket RPC error. Though that can come over time.

Reply:
totally. At least this gives clear feedback. We can workshop the friendly error format as time goes on 😄